### ISSUE TITLE
Restructure project folders

### ISSUE THREAD

**Issue by reyraa** [LiskHQ/lisk-hub#2287]
### Description
In order to reduce complexity and improve the project structure, the components folder should get rearranged into two main directories: one for screens and another one for shared components.

This proposal shows how to rearrange the folder: https://www.notion.so/lightcurve/Component-restructure-Proposal-57352eacbc484856931488cada892c09

### Acceptance criteria
 - The components directory should have 2 directories within: `screens` and `shared`.
 - Router components must be located in `screens` directory.
 - Every component which is used in more than one screen must be in `shared`.
 - Every component which is directly referred by a route must be in `screens`.
-  Routes must correspond to the directory name of the component they refer to.

**Comment by slaweet** [comment]
@reyraa Is this duplicate to https://github.com/LiskHQ/lisk-hub/issues/2194 ?

**Comment by slaweet** [comment]
I would propose one more requirement (can be also a separate ticket):
- The name of each subfolder in `src/components/screens` is exactly the same as the route it corresponds to. E.g. http://localhost:8080/#/wallet component is in `src/components/screens/wallet/index.js`; http://localhost:8080/#/wallet/send is in `src/components/screens/wallet/send/index.js`; http://localhost:8080/#/bookmarks/add-bookmark is in `src/components/screens/bookmarks/add-bookmark/index.js`

Only after that change is the change worth. Otherwise, I still have to go to `/src/constants/routes.js` to find the component based on my route, so it IMO doesn't matter that much if the component is in `src/components/screens` or `src/components/`

**Comment by reyraa** [comment]
@slaweet Thank you. I closed #2194 and added this rules.
